dist/tools/insufficient_memory/add_insufficient_memory_board.sh

```diff
@@ -30,21 +32,25 @@ else
     CRESET=
 fi
 
+if [ "$1" == "--no-docker" ]; then
+    shift 1
+else
+    # Use a standardized build within Docker and with minimal output
+    export DOCKER_MAKE_ARGS=${MAKE_ARGS}
+    export RIOT_CI_BUILD=1
+    export BUILD_IN_DOCKER=1
+fi
+
 BOARD=$1
 RIOTBASE=$(dirname $0)/../../..
 APPLICATIONS+=examples/*/Makefile
 APPLICATIONS+=" "
 APPLICATIONS+=tests/*/Makefile
 
-# Use a standardized build within Docker and with minimal output
-export BUILD_IN_DOCKER=1
-export DOCKER_MAKE_ARGS="-j4"
-export RIOT_CI_BUILD=1
-
 for app in ${APPLICATIONS}; do
     application=$(dirname ${app})
     printf "${CNORMAL}%-40s${CRESET}" ${application}
-    output=$(make BOARD=${BOARD} -C ${RIOTBASE}/${application} 2>&1)
+    output=$(make BOARD=${BOARD} ${MAKE_ARGS} -C ${RIOTBASE}/${application} 2>&1)
```

Comment:
When using docker, `MAKE_ARGS` is added twice in the command line (here and via `DOCKER_MAKE_ARGS`). You could have a third variable, used in the command line that is set to `MAKE_ARGS` without docker, and empty in docker.

Reply:
I added a `jobs` variable